src/components/BeforeAndAfter/BeforeAndAfter.module.scss

```diff
@@ -0,0 +1,46 @@
+@import 'scss/global.scss';
+
+.BeforeAndAfter {
+    display: flex;
+    justify-content: center;
+    background-color: $teriary-background-color;
+    min-height: 160px;
+    color: $primary-text;
+}
+
+.Image {
+    align-self: center;
```

Comment:
Not sure what the difference between this and `align-items: center` is, but if they're the same, let's stick to `align-items`. Up to you though!